proto/permit.proto

```diff
@@ -11,11 +11,12 @@ service permit {
 
 message CreatePermitRequest {
     string fileID = 1;
-    string sharerID = 2;
-    repeated User users = 3;
-    string classification = 4;
-    string info = 5;
-    repeated string approvers = 6;
+    string fileName = 2;
```

Comment:
dont change the numbers. its unnecessary breaking changes. add fileName in the end